cosmwasm/ibc-union/core/src/contract.rs

```diff
@@ -624,6 +625,19 @@ pub fn execute(
                     .add_attribute("relayer", relayer),
             ))
         }
+        ExecuteMsg::CommitClientStatus(msg) => {
+            let status = query_light_client::<Status>(
+                deps.as_ref(),
+                client_impl(deps.as_ref(), msg.client_id)?,
+                LightClientQuery::GetStatus {
+                    client_id: msg.client_id,
+                },
+            )?;
+            let encoded = U256::from(status as u32);
+            deps.storage
```

Comment:
this needs to be written to the commitment store, *not* the regular storage. it needs to be committed as the un-hashed value. use `store_commit`.